server/util/src/logger.rs

```diff
@@ -0,0 +1,19 @@
+use std::env;
+
+pub enum LogLevel {
+    Info,
+    Warn,
+}
+pub fn init_logger(level: LogLevel) {
```

Comment:
would be helpful to add some instructions to the readme about configuring the logging and allowable log levels

Reply:
Thanks Mark, i've added an issue for now: https://github.com/openmsupply/open-msupply/issues/new